test/integration/mon/serverspec/server_spec.rb

```diff
@@ -2,6 +2,14 @@
 
 set :backend, :exec
 
+describe package('nagios-plugins-http') do
+  it { should be_installed }
+end
+
+describe file('/etc/nagios/nrpe.d/check_http.cfg') do
+  it { should exist }
+end
+
```

Comment:
Why do we need theses tests if `osl-nrpe::check_http` already has serverspec tests?